mission-control/docs/playbooks/Actions/gitops.mdx

```diff
@@ -120,6 +120,43 @@ Files create or delete existing files on the git repo.
 | `path`    | Path to file                                                              | `string` | `true`   | `true`      |
 | `content` | Content of the file. Use the `$delete` keyword to delete an existing file | `string` | `true`   | `true`      |
 
+## Env Vars
+
+GitOps action receives a set of commonly used [env vars](../../reference/playbooks/env-vars). These env vars are extracted by traversing up the Flux Kustomization & Git Repository that created the config resource the playbook is running against. 
+They are
+
+| Field       | description                            |
```

Comment:
I think we can combine the 3 tables e.g. `git.file`

Reply:
Also update the example to use a fix repo / file path and move the env section to a detail show the equivalent without relying on the helpers.